gpuarray/config/gen_types.py

```diff
@@ -45,9 +46,9 @@ def add_type(name, C, sz):
 typedef unsigned __int32 uint32_t;
 typedef signed __int64 int64_t;
 typedef unsigned __int64 uint64_t;
-#else
+#else  // _MSC_VER
```

Comment:
Is this specific for `#else` or for `#endif` too? What style is advised in general?